pom.xml

```diff
@@ -7,6 +7,18 @@
     <groupId>io.bittiger</groupId>
     <artifactId>ads-searching-system</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <build>
```

Comment:
This is awesome! Thanks for bringing the maven compiler in!
You can put the `build` section after the `properties` section and put your version number (i.e. 1.6) in the properties section.
Sometimes the version is used by multiple modules so that properties section can keep all the dependencies in consistency.